pkg/cloud/mesos/config.go

```diff
@@ -0,0 +1,79 @@
+package mesos
+
+import (
+	"io"
+	"time"
+
+	"code.google.com/p/gcfg"
+)
+
+const (
+	DefaultMesosMaster       = "localhost:5050"
+	DefaultHttpClientTimeout = time.Duration(10) * time.Second
+	DefaultStateCacheTTL     = time.Duration(5) * time.Second
+)
+
+var (
+	config *Config
+)
+
+func getConfig() *Config {
+	return config
+}
+
+func resetConfigToDefault() {
+	config = createDefaultConfig()
+}
+
+func init() {
+	resetConfigToDefault()
+}
+
+// Example Mesos cloud provider configuration file:
+//
+// [mesos-cloud]
+//  mesos-master        = leader.mesos:5050
+//	http-client-timeout = 500ms
+//	state-cache-ttl     = 1h
+
+type ConfigWrapper struct {
+	Mesos_Cloud Config
+}
+
+type Config struct {
+	MesosMaster            string          `gcfg:"mesos-master"`
+	MesosHttpClientTimeout WrappedDuration `gcfg:"http-client-timeout"`
+	StateCacheTTL          WrappedDuration `gcfg:"state-cache-ttl"`
+}
+
+type WrappedDuration struct {
```

Comment:
it looks like @sttts wants the same thing as part of #246. should probably put this is a common place. also, should probably use an anonymous member here instead. for an example, see how k8s wraps Time: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/util/time.go